plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/META-INF/MANIFEST.MF

```diff
@@ -37,5 +46,7 @@ Import-Package: com.google.cloud.tools.eclipse.ui.util,
  org.eclipse.swt.graphics,
```

Comment:
do we still need any of these swt imports since the ui code is in a different package?